server/ms_citizen/README.md

```diff
@@ -1,4 +1,4 @@
-## This folder will contain the citizen microservice containing user verification code.
+## This folder contains the citizen microservice containing user verification code.
 The Port Number used by the Verification is : 5001
 
 The Route used is : /verify
```

Comment:
can u also add request format? as a json object with the data types the route accepts.
my example:
### POST /code

#### Updates the EFTS code for the TC Kimlik Number (id)

#### Request Format
{
    id: number,
    eftsCode: string
}

#### Respose Format On Success
{
    msg: "Code Added."
    updatedNode: [Node Object] 
}

Reply:
Ok I will add that request format as well